#! /bin/sh /usr/share/dpatch/dpatch-run
## -*- diff -*-
## 50_rename-octave-tags.dpatch by Rafael Laboissiere <rafael@debian.org>
##
## DP: Rename otags to be octave-tags, to avoid conflict with the Ocaml otags
## DP: package.  These changes are already in upstream CVS.

@DPATCH@

--- octave2.1-2.1.73.orig/doc/interpreter/emacs.txi
+++ octave2.1-2.1.73/doc/interpreter/emacs.txi
@@ -339,9 +339,9 @@
 
 @cindex TAGS
 @cindex Emacs TAGS files
-@cindex @code{otags}
+@cindex @code{octave-tags}
 You can generate TAGS files for Emacs from Octave @file{.m} files using
-the shell script @code{otags} that is installed alongside your copy of
+the shell script @code{octave-tags} that is installed alongside your copy of
 Octave.
 
 Customization of Octave mode can be performed by modification of the
--- octave2.1-2.1.73.orig/emacs/otags
+++ octave2.1-2.1.73/emacs/otags
@@ -2,7 +2,7 @@
 
 # Generate a TAGS file from a set of Octave .m files for use with Emacs.
 #
-# Run as '$ otags' in the given Octave directory to generate a
+# Run as '$ octave-tags' in the given Octave directory to generate a
 # TAGS file.  If you want to include another directory, add a line
 # prior to the "*.m" line containing something like
 #  `--include=/path/to/other/directory/TAGS" \'.
--- octave2.1-2.1.73.orig/emacs/otags.1
+++ octave2.1-2.1.73/emacs/otags.1
@@ -1,19 +1,19 @@
 .\" Man page contributed by Dirk Eddelbuettel <edd@debian.org>
 .\" and released under the GNU GPL
-.TH OTAGS 1 "31 October 2001" "GNU Octave"
+.TH OCTAVE-TAGS 1 "31 October 2001" "GNU Octave"
 .SH NAME
-OTAGS - Generate Emacs tags file from GNU Octave code
+OCTAVE-TAGS - Generate Emacs tags file from GNU Octave code
 .SH SYNOPSIS
-.BR otags\  [--include\ dir]
+.BR octave-tags\  [--include\ dir]
 \fIfile\fP .\|.\|.  
 .SH DESCRIPTION
 .PP
-.B otags
+.B octave-tags
 program is used to create a tag table file, in a format understood by 
 .BR emacs (1)
 and
 .BR xemacs (1).
-.B otags
+.B octave-tags
 reads the files specified on the command line, and write a tag table  
 (defaults: `TAGS') in the current working directory. Files specified with
 relative file names will be  recorded in the tag table with file names
